app/pages/Login.tsx

```diff
@@ -1,10 +1,232 @@
-import { View, Text } from 'react-native'
-import React from 'react'
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { useNavigation } from "@react-navigation/native";
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import {
+  ActivityIndicator,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  TextInput,
+} from "react-native";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { FirebaseError } from "firebase/app";
+import { AppDispatch, RootState } from "../store";
+import { setErrorMessage } from "../reducer/userSlice";
+import { auth } from "../config/firbase.config";
+import PasswordInput from "../components/PasswordInput";
+
+type RootStackParamList = {
```

Comment:
i think this RootStackParamList is repeated in register page too @kaushaki-kumari